src/app/Topology/Shared/Entity/EntityKeyValues.tsx

```diff
@@ -17,22 +17,31 @@ import { Label, LabelGroup } from '@patternfly/react-core';
 import * as React from 'react';
 import { EmptyText } from '../EmptyText';
 
-export const EntityLabels: React.FC<{ labels?: object; maxDisplay?: number }> = ({ labels, maxDisplay, ...props }) => {
-  const _transformedLabels = React.useMemo(() => {
-    return labels ? Object.keys(labels).map((k) => `${k}=${labels[k]}`) : [];
-  }, [labels]);
+export function keyValueEntryTransformer(kv: string[] | object): string[] {
+  return kv ? Object.keys(kv).map((k) => `${k}=${kv[k]}`) : [];
+}
 
-  return _transformedLabels.length ? (
-    <div className="entity-overview__displayed-labels-wrapper" {...props}>
+export function valuesEntryTransformer(kv: string[] | object): string[] {
```

Comment:
We probably don't need this: https://github.com/cryostatio/cryostat-web/pull/1111/files#r1329154630